event/listener.php

```diff
@@ -65,7 +65,11 @@ static public function getSubscribedEvents()
 	*/
 	public function load_google_analytics()
 	{
-		$this->template->assign_var('GOOGLEANALYTICS_ID', $this->config['googleanalytics_id']);
+		$this->template->assign_vars(array(
+			'GOOGLEANALYTICS_ID'		=> $this->config['googleanalytics_id'],
+			'GOOGLEANALYTICS_USER_ID'	=> $this->user->data['user_id'],
+			'S_GOOGLEANALYTICS_USER_ID'	=> $this->user->data['is_registered'],
```

Comment:
We have S_REGISTERED_USER but nothing with just the user ID, although we have SESSION_ID.